pkg/spanconfig/spanconfigkvsubscriber/testdata/system_span_configs

```diff
@@ -0,0 +1,127 @@
+
+start
+----
+
+update
+upsert [a,c):A
+upsert [d,f):D
+----
+
+updates
+----
+/M{in-ax}
+[a,c)
+[d,f)
+
+store-reader key=a
+----
+conf=A
+
+store-reader key=e
+----
+conf=D
+
+store-reader key=g
+----
+conf=MISSING
+
+update
+upsert {entire-keyspace}:_CLUSTER
+----
+
+updates
+----
+/M{in-ax}
+
+store-reader key=a
+----
+conf=A_CLUSTER
+
+store-reader key=e
+----
+conf=D_CLUSTER
```

Comment:
Instead of concatenating and making use of _'s, what if this syntax captured exactly what was being appended? 
```
conf=D+X
```